packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -165,6 +166,47 @@ export const useArbTokenBridge = (
     updateEthBalances()
   }
 
+  const depositEthFromContract = async (weiValue: BigNumber) => {
+    const etherVal = utils.formatUnits(weiValue, 18)
+
+    const inboxAddress = bridge.l1Bridge.network.ethBridge?.inbox;
+    if(!inboxAddress) throw new Error("Inbox address not found")
+
+    const walletAddress = await walletAddressCached()
+    const inbox =  Inbox__factory.connect(inboxAddress, bridge.l1Signer)
+    const maxSubmissionPrice = (
+      await bridge.l2Bridge.getTxnSubmissionPrice(0)
+    )[0].mul(4)
```

Comment:
I think we could have an arb-ts utility function that we pass in the result of `bridge.getRetryableTxnParams(` and it auto does these bump ups